examples/03-03_os-eol.mjs

```diff
@@ -0,0 +1,12 @@
+const os = require('os');
+const fs = require('fs');
+
+const text = 'This is the first line' + os.EOL + 'This is the second line';
+
+fs.writeFile('example.txt', text, (err) => {
```

Comment:
Hi in the example, I would avoid creating files, unless is strictly required by the function we are going to example.
Just to avoid that the readers run the examples and seeing something unexpected (like new files), and to avoid introducing more import. I prefer to import only os because is required by EOL.

In the Markdown is ok to have  this example (maybe we can have more examples, the first one prints to the console the string that includes EOL, and the second example with a writeFile)